app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        raise ValueError("Command should have three arguments")
```

Comment:
separate command by variables after this check to improve readability
f.e. `some_name1, some_name2, some_name3 = command`
And use it further